models/traitValue.tsx

```diff
@@ -18,7 +18,7 @@ import { Traits } from "./trait";
 export default interface TraitValue {
   id: string;
   name: string;
-  rarity: number;
+  rarity: number | string; // string to allow . or trailing 0s in inline editing
```

Comment:
could we convert to number before storing in the model?

Reply:
Yes! This should actually be taken care of already on line 90 of ```components/TraitValuesRow/index.tsx```
![image](https://user-images.githubusercontent.com/21000756/158089777-19aa80e1-3a08-4a33-82c5-9a2bed14bdfb.png)

Reply:
ok, can we revert the change here then?